Automattic/sensei
# Add/video block hook provide block dom

Fixes [sensei-pro#1204](1204-gh-Automattic/sensei-pro)

### Changes proposed in this Pull Request

* Adds support for requiring a video for lesson completion for individual blocks via block attribute.
* Provides video block DOM element to the video action `sensei.videoProgression.registerVideo` hook.

### Testing instructions


* Checkout the `add/video-block-hook-provide-block-dom` branch and build the assets.
* Confirm all the functionality of the Course Video Progression works as before and there are no issues.


### New/Updated Hooks

* Action Hooks
  * `sensei.videoProgression.registerVideo`
    * Now provides additional property that is a DOM element reference of the video block.
